urls.py

```diff
@@ -24,6 +29,15 @@
 admin.site.site_header = admin_header
 admin.site.site_title = admin_header
 
+SITEMAPS_CACHE_TTL = 24 * 60 * 60  # 24 Hours
```

Comment:
Let's drop this to 12 hours and see what it looks like. I have been bitten by 24 caching + once-a-day check, meaning ~2-day lag. 

```suggestion
SITEMAPS_CACHE_TTL = 12 * 60 * 60  # 24 Hours
```